specification/batch/Azure.Batch/routes.tsp

```diff
@@ -17,17 +17,9 @@ namespace Azure.Batch;
 
 // Interfaces ////////////////////
 
-#suppress "@azure-tools/typespec-azure-core/no-response-body" "The body of non-204 responses should not be empty."
-#suppress "@azure-tools/typespec-azure-core/no-error-status-codes" "Azure REST API guidelines recommend using 'default' error response for all error cases. Avoid defining custom 4xx or 5xx error cases."
-#suppress "@azure-tools/typespec-azure-core/no-rpc-path-params" "Operations defined using BatchOperation should not have path parameters."
-#suppress "@azure-tools/typespec-azure-core/casing-style" "The names of Model types must use camelCase"
-#suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Operation 'Exists' should be defined using a signature from the Azure.Core namespace."
-#suppress "@azure-tools/typespec-azure-core/rpc-operation-request-body" "BatchOperation with '@get' cannot have a body."
-@client({
-  name: "BatchClient",
-  service: Azure.Batch,
-})
-interface Batch {
+#suppress "@azure-tools/typespec-azure-core/no-rpc-path-params" ""
+@tag("Application")
+interface Application {
```

Comment:
I think the common naming convention is to pluralize these interface names (so "Applications" instead of "Application"). A quick search through this repo seems to confirm that.

Reply:
Also, I believe the tags should be plural as well